docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -10,5 +10,30 @@ There you can find links to upgrade notes for other versions too.
 - *(low priority)* add `product-search-export-products` as a dependency of `build-demo` phing target in your `build.xml`
 if you want to have products data exported to Elasticsearch after `build-demo` target is run ([#824](https://github.com/shopsys/shopsys/pull/824/files))
 
+### Application
+- moved data fixtures to project-base [#854](https://github.com/shopsys/shopsys/pull/854)
+    - moving data fixtures was done in this [#854](https://github.com/shopsys/shopsys/pull/854) so you can get inspiration how to do it
+        - you need to move all data fixtures to a new folder `DataFixtures` in `ShopBundle`
```

Comment:
OK so from where to where

Reply:
I removed this line because it's said in the previous point